README.md

```diff
@@ -14,14 +14,14 @@
 | Javascript   | [sdk available](https://github.com/TheThingsNetwork/node-app-sdk) | **core team** |
 | Objective-C  | nothing yet; help wanted | you? |
 | PHP          | files generated; not tested; help wanted | you? |
-| Python       | files generated; not tested; help wanted | you? |
+| Python       | files generated; not tested ([#23](https://github.com/TheThingsNetwork/api/pull/23)) | **core team** |
```

Comment:
1. Will merge #23 soon, so let's not add a link to the PR.
2. Do we (you) want to maintain it on behalf of the core team? Otherwise we should keep `help wanted | you`